build.gradle

```diff
@@ -522,13 +522,6 @@ subprojects {
   // The suites are for running sets of tests in IDEs.
   // Gradle will run each test class, so we exclude the suites to avoid redundantly running the tests twice.
   def testsToExclude = ['**/*Suite.class']
-  // Exclude PowerMock tests when running with Java 16 or newer until a version of PowerMock that supports the relevant versions is released
```

Comment:
There's another comment about PowerMock at line 822, can we remove it as well?

Reply:
There are also comments about EasyMock in 4 places:
- 2 in `DescribeTopicPartitionsRequestHandlerTest`
- 2 in `KafkaApisTest`
Let's clean those up as well.

Reply:
Hi @mimaison, thanks for the review. I remove all remaining comments.